### PR TITLE
add aria-label to input

### PR DESCRIPTION
**Pull request by liorur**

### CODE REVIEW

File: src/Input/Input.driver.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const inputDriverFactory = ({element, wrapper, component}) => {
     getReadOnly: () => input.readOnly,
     getTextOverflow: () => input.style['text-overflow'],
     getType: () => input.type,
+    getProp: propName => input.getAttribute(propName),
```
Comment (shlomitc) [inline src/Input/Input.driver.js]: change to specific attribute (getAriaLabel, getAraiaControls)

File: src/Input/Input.js [hunk 2]
```diff
@@ -88,7 +90,8 @@ class Input extends Component {
         onKeyUp={onKeyUp}
         readOnly={readOnly}
         type={type}
-
+        {...(ariaLabel && {'aria-label': ariaLabel})}
```
Comment (shlomitc) [inline src/Input/Input.js]: Extract to an aria object and spread here

File: src/Input/InputSuffix.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ const InputSuffix = ({theme, errorMessage, error, disabled, help, helpMessage, o
     },
     {
       component: () =>
-        <div onClick={onClear} className={styles.clearButton}>
+        <div onClick={onClear} className={styles.clearButton} tabIndex={0}>
```
Comment (shlomitc) [inline src/Input/InputSuffix.js]: remove from the PR

**Comment by nirhart** [comment]
@liorur please see @shlomitc reviews

File: src/Input/InputSuffix.js (hunk 3 above)
Comment (Faradey27) [inline src/Input/InputSuffix.js]: @shlomitc  why tabIndex should be removed? can it be optional and passed via props?

Comment (shlomitc) [inline src/Input/InputSuffix.js]: It's an OK addition, just not in this PR. should be a separate one

Comment (Faradey27) [inline src/Input/InputSuffix.js]: thank you, understood

**Pull request merged**